src/context/WalletProvider/NativeWallet/components/LegacyLogin.tsx

```diff
@@ -41,36 +36,70 @@ export const LegacyLogin = () => {
     handleSubmit,
     register,
     formState: { errors, isSubmitting },
+    reset,
   } = useForm({ shouldUnregister: true })
 
   const translate = useTranslate()
 
-  const isLoginError = (err: any): err is LoginError => err && typeof err.message === 'string'
+  const isLoginError = (err: any): err is LoginError =>
```

Comment:
Ditto here - argument was previously typed as `any` in this guard but we should be able to assume a generic `Error` type

Reply:
type Error doesn't have `response` as a property. Since LoginError is more of an extension of Error, if we type the parameter as Error, we can't check for properties to assert the type predicate `err is LoginError`.